requirements.txt

```diff
@@ -1,6 +1,6 @@
 cosl>=0.0.12
 # pinned to 2.16 as 2.17 breaks our unittests
-ops == 2.16
```

Comment:
This pin was removed by updating the unit tests and removing technical debt. This pin is incompatible with Scenario v7